coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -4767,6 +4767,30 @@ def test_flatten(self, compute_unit, backend, start_dim, end_dim, is_dynamic):
         )
 
 
+class TestUnflatten(TorchBaseTest):
+    @pytest.mark.parametrize(
+        "compute_unit, backend",
+        itertools.product(compute_units, backends),
+    )
+    def test_unflatten(self, compute_unit, backend):
+        class Head(nn.Module):
+            def __init__(self, nhead, input_size, hidden_size):
+                super(Head, self).__init__()
+                self.nhead = nhead
+                self.linear = nn.Linear(nhead * input_size, nhead * hidden_size)
+                self.unflatten = nn.Unflatten(-1, (nhead, hidden_size))
```

Comment:
Can you extend the test converage by testing different values of `dim`?

Reply:
Extended as suggested